ask-sdk-servlet-support/tst/com/amazon/ask/servlet/verifiers/SkillRequestTimestampVerifierTest.java

```diff
@@ -98,6 +113,11 @@ public void verify_nullTimestamp_throws_exception() {
         verifier.verify(mockServletRequest, serializedRequestEnvelope, RequestEnvelope.builder().withRequest(IntentRequest.builder().build()).build());
     }
 
+    @Test (expected = IllegalArgumentException.class)
+    public void construct_withNullTimeUnit_throwsIllegalArgumentException() {
```

Comment:
Can we also add a test case that compares the behavior a verifier constructed with the existing, millisecond-only constructor with that of one constructed with a time unit? For instance, if they are both set for 2 seconds (2000ms/2) then verify that a request 2000ms old passes, but 2001 fails in both cases. Effectively this just ensures the unit conversion is done properly.